### DISCUSSION TITLE
forcing autoattack to attack players and other entites

### DISCUSSION THREAD

**Discussion by ghost** [ORelio/Minecraft-Console-Client#2005] [Q&A]
how can i do it? there is no available setting for it

**Comment by ReinforceZwei** [comment]
You cannot. See #1017

**Reply by ghost** [reply]
can i write and share my killaura script for players?

**Reply by ORelio** [reply]
we do not encourage cheating, so please don't share cheat-related bots here.
of course, we can't prevent you from writing custom bots for personal use.
